packages/markdown/components/Code/index.jsx

```diff
@@ -5,12 +5,35 @@ const PropTypes = require('prop-types');
 // apps. Necessary because of people like this:
 // https://github.com/codemirror/CodeMirror/issues/3701#issuecomment-164904534
 const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/syntax-highlighter') : false;
+const copy = require('copy-to-clipboard');
+
+const remapLang = {
+  c: 'cplusplus',
+  'c++': 'cplusplus',
+  cpp: 'cplusplus',
+  docker: 'dockerfile',
+  html: 'htmlmixed',
+  js: 'javascript',
+  py: 'python',
+  sh: 'shell',
+  bash: 'shell',
+  mysql: 'sql',
+};
 
 function Code(props) {
   const { className, children, lang, meta } = props;
-  const language = (className || '').replace('language-', '');
+
+  const langClass = className.search(/lang(?:uage)?-\w+/) >= 0 ? className.match(/\s?lang(?:uage)?-(\w+)/)[1] : '';
+  const language = lang in remapLang ? remapLang[lang] : lang || langClass;
+  console.log({ langClass, language });
```

Comment:
Left in debug

Reply:
Good catch; this should be resolved in the [rdmd/next](https://github.com/readmeio/api-explorer/pull/632) branch!